src/SimplCommerce.WebHost/Program.cs

```diff
@@ -73,8 +73,7 @@ void ConfigureService()
         }
     }
 
-    builder.Services.AddScoped<ServiceFactory>(p => p.GetService);
-    builder.Services.AddScoped<IMediator, Mediator>();
+    builder.Services.AddMediatR(cfg => cfg.RegisterServicesFromAssembly(typeof(Program).Assembly));
```

Comment:
Could you please check any service that is using `MediatR` to make sure everything is working as expected